src/Endpoints/Delegates/HandlesSettings.php

```diff
@@ -275,4 +275,28 @@ public function resetTypoTolerance(): array
     {
         return $this->http->delete(self::PATH.'/'.$this->uid.'/settings/typo-tolerance');
     }
+
+
+    // Settings - Word dictionary
+
+    /**
+     * @return list<non-empty-string>
+     */
+    public function getWordDictionary(): array
+    {
+        return $this->http->get(self::PATH.'/'.$this->uid.'/settings/dictionary');
+    }
+
+    /**
+     * @param list<non-empty-string> $wordDictionary
+     */
+    public function updateWordDictionary(array $wordDictionary): array
```

Comment:
Can we typehint returned array structure?

Reply:
Yes, I thought about it, but all setting updates return a task, and none are specified, so I didn't put anything in.

Reply:
Then you could declare the Task return type in some file and then import the type where a task is returned,but then maybe in a separate PR :)

Reply:
I will create an issue about it

Reply:
An alternative could be to introduce some value object that would hold the task info